spec/controllers/idv/cancellations_controller_spec.rb

```diff
@@ -227,8 +227,7 @@
         before do
           allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
           allow(controller).to receive(:user_session).and_return(
-            'idv/in_person' => { 'pii_from_user' => {},
-                                 'Idv::Steps::InPerson::StateIdStep' => true },
+            'idv/in_person' => { 'pii_from_user' => {} },
```

Comment:
Ohh- I resolve the comment above. There were 2 instances of `Idv::Steps::InPerson::StateIdStep` in this file. You deleted one but the other should be deleted. See line 254 in the before do block for context `with in pending in-person enrollment`

Reply:
Good catch, thank you!!